### DISCUSSION TITLE
Setting up on AWS EC2 instance

### DISCUSSION THREAD

**Discussion by benjibee** [orangecoding/fredy#81] [Q&A] [answered]
I'm looking to get this up and running with an EC2 instance. So far I've setup the instance, SSH'd into it, and used `yum` to install all required packages and then run `yarn install && yarn prod && yarn start` without any errors.

The issue is if attempt to access it using port `:9998` I always get a 404 error. If I stop `yarn start` and try again I get an error that the server isn't responding, so I know it's getting the requests. I've tested it in my browser and using `curl -I http://localhost:9998` when SSH'd into the instance, both get the same response.

I've tried messing with the Vite config file setting the host to the private IP of the instance to no avail. I also noticed if I change the port to something like `:8000` in the config, it still shows `:9998` in the console so I feel like I'm doing something quite wrong here.

The root directory for the project and its files is the home directory for my instance's user: `/home/ec2-user` if that helps.

**Comment by orangecoding** [comment, marked as answer]
Hey.
If you want to change the port, you typically do that in your config.json or through the UI. For your specific situation, it might make sense to consider using the Docker package. I'm not rly  familiar with EC2. I do know that many people have deployed Fredy to Vercel.

**Reply by benjibee** [reply]
Thanks for the reply. I tried a few times after this without much progress. I finally just ran the docker file and setup a reverse proxy for the port, it's working just fine now.
